collector/hwmon_linux.go

```diff
@@ -351,6 +372,27 @@ func (c *hwMonCollector) hwmonName(dir string) (string, error) {
 	return "", errors.New("Could not derive a monitoring name for " + dir)
 }
 
+func (c *hwMonCollector) hwmonHumanReadableChipName(dir string) (string, error) {
+	// this is similar to the methods in hwmonName, but with different
```

Comment:
Can you move this above the function and start with "hwmonHumanReadableChipName is similar to ...."?